Src/PerformanceCollector/Unit.Tests/QuickPulse/QuickPulseDataSampleTests.cs

```diff
@@ -11,10 +11,12 @@
     [TestClass]
     public class QuickPulseDataSampleTests
     {
+        private readonly DateTimeOffset now = new DateTimeOffset(2017, 1, 1, 0, 0, 0, 0, TimeSpan.Zero);
```

Comment:
nitpick: I suggest calling it fakeNow so no one gets confused in future thinking it is really "now".